CorsixTH/Lua/research_department.lua

```diff
@@ -478,6 +478,17 @@ function ResearchDepartment:improveMachine(machine)
     research_info.start_strength = research_info.start_strength
       + improve_rate
     research_info.strength_imp = research_info.strength_imp + 1
+    if self.world.ui.app.config.new_machine_extra_info then
+      -- Update dynamic info for any machines that now have improved strength.
+      for _, room in ipairs(self.world.rooms) do
+        for obj in pairs(room.objects) do
+          if(obj.object_type.id == machine.id) then
```

Comment:
```suggestion
          if (obj.object_type.id == machine.id) then
```

Reply:
LOL - those brackets are a product of years of habit - fixed now. Cheers.